docs/network_info_plus/usage.mdx

```diff
@@ -27,6 +27,8 @@ To successfully get WiFi Name or Wi-Fi BSSID starting with Android O, ensure all
 
  * Location services are enabled on the device (under Settings > Location).
 
+ * If you use device with Android 12 (API level 31) and newer be sure that your app has ACCESS_NETWORK_STATE permission.
```

Comment:
Would this requirement break existing apps when devs don't add the permissions to their apps? Or will it simply silently fail to obtain the IPs?

Reply:
ah you mention this already: but during testing on emulators code worked fine for me even without this permission being added into AndroidManifest.

Reply:
Yes, it is quite strange that API docs state the requirement of it, while on API 33 I have no issues with getting IPs. Hope it is not different on real devices.